docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 ---
-title: Getting Started With Access Controls
-description: Get started using Teleport Access Controls.
+title: Getting Started With Role-Based Access Control
+description: Teleport Role-Based Access Control.
 ---
 
-In Teleport, any local, SSO, or robot user can be assigned one or several roles.
-Roles govern access to databases, SSH servers, Kubernetes clusters, Windows
-desktops, and web apps.
+In Teleport, RBAC (Role-Based Access Control) is a system that helps you manage who can access various resources such as databases, SSH servers, Kubernetes clusters, Windows desktops, and web apps. 
 
-We will start with local users and preset roles, assign roles to SSO users, and
-wrap up with creating your own role.
+## What is RBAC in Teleport?
 
-## Prerequisites
+In Teleport, any local, SSO, or robot user can be assigned one or several roles. Roles govern access to your resources and infrastructure.
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+Each role defines specific permissions to ensure that users only have access to what they need. 
 
-- (!docs/pages/includes/tctl.mdx!)
+## What is a role?
 
-(!docs/pages/includes/permission-warning.mdx!)
+Roles determine what resources a user can access and what actions they can perform within the system. 
 
-## Step 1/3. Add local users with preset roles
+By creating and assigning roles, administrators can enforce security policies, manage access, and ensure users have the appropriate level of privileges based on their responsibilities.
 
-Teleport provides several preset roles:
+## Teleport users
```

Comment:
Would it make sense to focus this guide on roles and label matching and, rather than spell out the kinds of users here, link to the [User Types](https://pr-48029.d3pp5qlev8mo18.amplifyapp.com/docs/reference/user-types) reference in a "Further reading" or "Next steps" section?

Reply:
@mmcallister What do you think about the comment above? In general, I think this guide should be a lot more focused. I would start by identifying the objective of the guide in the intro paragraph (making it as narrow as possible), then editing the structure of the guide based on that objective. How does that sound?